02-peer-protocol.md

```diff
@@ -602,7 +602,7 @@ The `shutdown` response requirement implies that the node sends `commitment_sign
 
 ### Closing Negotiation: `closing_signed`
 
-Once shutdown is complete and the channel is empty of HTLCs, the final
+Once shutdown is complete, the channel is empty of HTLCs, there are no dangling commitments and all updates are included on both commitments, the final
```

Comment:
```suggestion
Once shutdown is complete, the channel is empty of HTLCs, there are no dangling 
commitments and all updates are included on both commitments, the final
```

The same comment of Matt here, be more specific it is good here because we avoid inconsistency for future updates or ambiguity.